### DISCUSSION TITLE
Copyright Headers

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#3881] [Ideas]
@opencollective/core : I just had this idea while working on another projects copyright notices. So, I was wondering if we should insert a copyright header notice on all files in OpenCollective (at least in the main reops such as the front end and api)? 🤔

Reference: https://reuse.software/spec/

**Comment by Betree** [comment]
That would be a tedious practice to enforce, and a big change to enforce it for existing files. I understand that it can be needed for some projects (ie. libraries with advanced algorithms) but our code is so tied to Open Collective's business that I don't think we need to be too protective with it. Are there arguments in favor of this practice that I'm missing?

**Reply by SudharakaP** [reply]
Agreed. If it's not protective and copying parts of the code isn't a big issue I think it's fine. The spec I referred above seems to recommend that practice; but as you said it's probably more relevant to libraries and stuff which has specific algorithms etc. 😄
